<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(213, 213, 213);
border-radius:10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_superior">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>801</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(158, 158, 158);
border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="bt_close_boca">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>10</y>
          <width>36</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>iconos/cerrar.svg</normaloff>iconos/cerrar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>471</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Cantarell</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left:20px;
margin-top:8px;
color: #FFFFFF;</string>
        </property>
        <property name="plainText">
         <string>Boca</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bt_minimize_boca">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>10</y>
          <width>36</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>iconos/minimizar.svg</normaloff>iconos/minimizar.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_slider_boca">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>52</y>
         <width>461</width>
         <height>78</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Boca</string>
        </property>
       </widget>
       <widget class="QSlider" name="bocaSlider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>240</width>
          <height>41</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal
{
    border: 1px solid #5d9eff;
    height: 10px;
	background: #5d9eff;
}  

QSlider::handle:horizontal
{
    background: #5d9eff;;
    width: 30px;
	height: 30px;
	left: 11px;
	right: 11px;
	margin: -12px;
	border-radius: 12px;	
}

QSlider::add-page:horizontal
{
    background-color: #9f9f9f;
	border: 1px solid #9f9f9f;
}</string>
        </property>
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
        <property name="tickInterval">
         <number>0</number>
        </property>
       </widget>
       <widget class="QLabel" name="boca_valor">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>20</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaBocaMinimo">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>21</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaBocaMedio">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>50</y>
          <width>31</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaBocaMaximo">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>50</y>
          <width>31</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaBocaPin">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-left: 1px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_OnOff_boca">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>52</y>
         <width>120</width>
         <height>78</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="bocaOnOff">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>ArialBlack</family>
          <pointsize>12</pointsize>
          <weight>1</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	color: #2a2a2a;
	font: 87 14 pt &quot;ArialBlack&quot;;
}

QRadioButton::indicator
{
	width: 50px;
	height: 50px;
	border-radius: 25px;
}

QRadioButton::indicator:checked {
   background-color: #00ff00;

}

QRadioButton::indicator:unchecked 
{
	background-color: #da0037;
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaBocaOnOff">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 6px;
color: red;</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Motor</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_Bucle_boca">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>52</y>
         <width>120</width>
         <height>78</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QRadioButton" name="bocaBucle">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>ArialBlack</family>
          <pointsize>12</pointsize>
          <weight>1</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	color: #2a2a2a;
	font: 87 14 pt &quot;ArialBlack&quot;;
}

QRadioButton::indicator
{
	width: 50px;
	height: 50px;
	border-radius: 25px;
}

QRadioButton::indicator:checked {
   background-color: #00ff00;

}

QRadioButton::indicator:unchecked 
{
	background-color: #da0037;
	
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="etiquetaBocaBucle">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>30</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 6px;
color: red;</string>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>10</y>
          <width>51</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Bucle</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_Velocidad_boca">
       <property name="geometry">
        <rect>
         <x>700</x>
         <y>52</y>
         <width>51</width>
         <height>78</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QSpinBox" name="bocaSpinBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>51</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox::up-button { 
	width: 25px; 
	height: 25px; 
	background: none;
}
QSpinBox::down-button {
	top: -3px;
	width: 25px; 
	height: 25px;
	background: none;
}
QSpinBox { 
	padding-left: 6px;
}

</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Veloc.</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_Inicio_boca">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>52</y>
         <width>41</width>
         <height>78</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>41</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Inicio</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bocaPushButton">
        <property name="geometry">
         <rect>
          <x>1</x>
          <y>36</y>
          <width>41</width>
          <height>30</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(149, 149, 149);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Inicio</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>400</y>
         <width>340</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(0, 0, 0, 0);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="bocaProgressBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>341</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>PibotoLt</family>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(88, 88, 88);
color: rgb(248, 248, 248);</string>
        </property>
        <property name="text">
         <string> Espere un momento por favor</string>
        </property>
        <property name="openExternalLinks">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_voz">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>200</y>
         <width>461</width>
         <height>80</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButtonVoz">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-1</y>
          <width>461</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>23</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:rgb(149, 149, 149);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Prueba de Voz</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
